src-packed/textAnalytics.js

```diff
@@ -98,6 +98,12 @@ function replaceTrailingPunctuation (text) {
 export function preprocessText (text) {
     var result = removeCodeBlocks (text);
     result = removeImageTags (result);
+
+    // NOTE: ".NET" is not properly split into sentences.
+    // Because MSFT employees use ".NET" a lot in comments, we should fix it.
+    // Let's replace it with " NET", so text will be the same length.
+    result = result.replace ('.NET', ' NET');
```

Comment:
Does this handle different casings (".net", ".Net") too?

Reply:
Whoops, probably not. I would need regex syntax like `/\.NET/i`.